terraform/aws/providers.tf

```diff
@@ -0,0 +1,40 @@
+terraform {
+  required_providers {
+    databricks = {
+      source  = "databrickslabs/databricks"
+      version = "~>0.4.0"
+    }
+    aws = {
+      source  = "hashicorp/aws"
+      version = "~>3.68.0"
+    }
+  }
+}
+
+provider "aws" {
+  region  = var.aws_region
+  profile = var.aws_profile
+
+  default_tags {
+    tags = {
+      Vendor = "Databricks Unity Catalog"
+    }
+  }
+}
+
+// initialize provider in "MWS" mode, to add users at account-level
+provider "databricks" {
```

Comment:
split it into two different modules - one module with databricks provider on accounts level and another module for the workspace itself. otherwise customers will keep forgetting to add aliases.